Day-6-7-8/src/App.js

```diff
@@ -1,18 +1,46 @@
+import React, { Suspense, useEffect, useState } from "react";
 import ReactDOM from "react-dom/client";
-import { createBrowserRouter, Outlet, RouterProvider } from "react-router-dom";
-import UserCard from "./components/UserCard";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+const UserCard = React.lazy(() => import("./components/UserCard"));
 import Header from "./components/Header";
 import data from "./data/userGithubDetails.json";
 import UserCardInfo from "./components/UserCardInfo";
 import { ErrorComponent } from "./components/ErrorComponent";
 import UserProfile from "./components/UserProfile";
```

Comment:
Can make separate folders for reusable and not-dummy components and through index.js of both folders can export(named) these components. Here there are so many imports, it's harder to debug and maintain.